forge-gui/res/cardsfolder/upcoming/infectious_bite.txt

```diff
@@ -0,0 +1,10 @@
+Name:Infectious Bite
+ManaCost:1 G
+Types:Instant
+A:SP$ Pump | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Choose target creature you control | RememberTargets$ True | SubAbility$ DBFight | StackDescription$ Target creature you control [{c:ThisTargetedCard}] | SpellDescription$ Target creature you control deals damage equal to its power to target creature you don't control. Each opponent gets a poison counter.
+SVar:DBFight:DB$ DealDamage| ValidTgts$ Creature.YouDontCtrl | TgtPrompt$ Choose target creature you don't control | NumDmg$ X | SubAbility$ DBPoison 

```

Comment:
- Missing DamageSource
- Use ParentTargeted instead of remembering

Reply:
The Amount should still be "ParentTargeted$"

calculation uses a different path than defined